av/include/ignition/common/HWEncoder.hh

```diff
@@ -1,19 +0,0 @@
-/*
- * Copyright 2020 Martin Pecka
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- */
-
-#include <gz/common/HWEncoder.hh>
```

Comment:
why is this file deleted?

Reply:
`HWEncoder.hh` is a private header, and is prevented from installation in the CMakeLists here: https://github.com/gazebosim/gz-common/blob/b9859bdfb5f7db2839133b10ae7a7a3dfc6eadb9/av/include/gz/common/CMakeLists.txt#L1-L2

I think we don't need a corresponding ignition/common header for this reason.

Reply:
gotcha; this makes sense. In sdformat, we've moved non-installed headers to the `src` folder so that all headers in an `include` folder are installed, so I didn't think to check if it was excluded. No changes needed